nf_core/pipeline-template/subworkflows/local/generate_downstream_samplesheets/main.nf

```diff
@@ -0,0 +1,67 @@
+
+
+
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    SUBWORKFLOW SPECIFIC FOR RNASEQ
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+// TODO: Update the following workflow to a specific pipeline
+workflow SAMPLESHEET_RNASEQ {
+    take:
+    ch_reads
+    format
+
+    main:
+
+    ch_list_for_samplesheet = ch_reads.map { meta, reads ->
+        def out_path     = file(params.outdir).toString() + '/relative/custom/path/'
+        def sample       = meta.id
+        def fastq_1      = meta.single_end  ? out_path + reads.getName() : out_path + reads[0].getName()
+        def fastq_2      = !meta.single_end ? out_path + reads[1].getName() : ""
+        def strandedness = "auto"
+        [sample: sample, fastq_1: fastq_1, fastq_2: fastq_2, strandedness: strandedness]
+    }
```

Comment:
are we sure we want it to make so genomics specific?

Reply:
See reply above about pros/cons of example code (maybe with more TODOs.